wagtail/admin/viewsets/model.py

```diff
@@ -146,7 +171,20 @@ def get_history_view_kwargs(self, **kwargs):
         }
 
     def get_usage_view_kwargs(self, **kwargs):
-        return {**kwargs}
+        return {
+            "template_name": self.get_templates(
```

Comment:
Yeah, that seems fair to me. By its nature, a setting like that is only going to affect a single view, so it's more appropriate to override it on the view class rather than the viewset (and then pass the overridden view via `create_view_class` etc). I suspect that most of the time, you're not going to be changing the template without also modifying the view's `get_context` method anyway.